File: program-to-print-reciprocal-of-letters/program-to-print-reciprocal-of-letters.py
S="adoiw"
a = ""
for i in S:
    if i.isalpha():
        if i.isupper():
            sub = ord(i) - 65
            char = chr(90 - sub)
            a += char
        elif i.islower():
            sub = ord(i) - 97
            char = chr(122 - sub)
            a += char
    else:
        a+=i
print(a)

File: program-to-print-reciprocal-of-letters/program-to-print-reciprocal-of-letters.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){

    string S ="bdcauefoh";

    string str;
        char res;
        for(int i=0;i<S.length();i++)
        {
            if(islower(S[i]))
            {
                res = 'z' - S[i]+'a';
                str.push_back(res);
            }
            
            else if(isupper(S[i]))
            {
                res = 'Z' - S[i]+'A';
                str.push_back(res);
            }
            else 
                str += " ";
        }
        cout<<str;
}